### ISSUE TITLE
Convolutions don't match paper

### ISSUE THREAD

**Issue by rmcavoy** [signatrix/efficientdet#44]
According to the original paper (see quotes below), the convolutions in this implementation appear to be incorrect since among other reasons normal convolutions are used.
 With regard to the bifpn,
>  Notably, to further improve the efficiency, we use depthwise separable convolution [4, 29] for feature fusion (in the bifpn), and add batch normalization and activation after each convolution.

Thus we know that in the BIFPN EfficientDet uses depthwise separable convolutions and has batch norm and activation functions following each depthwise separable convolution rather than the normal convolutions used here.

> Batch normalization is added after
every convolution with batch norm decay 0.997 and epsilon 1e-4.

This directly states the all convolutions have batch norms but without stating whether depthwise separable convolutions are used outside the bifpn. It seems likely to me that all convolutions are depthwise separable because otherwise whichever convolution was not depthwise separable would be the bottleneck.

**Comment by zylo117** [comment]
You're right, I realized it too, so I implement a real efficientdet which follows the official efficientdet.
The result seems nice from D0 to D7.

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch
